src/main/java/org/opensearch/geospatial/action/upload/geojson/UploadGeoJSONTransportAction.java

```diff
@@ -47,6 +48,9 @@ public UploadGeoJSONTransportAction(
 
     @Override
     protected void doExecute(Task task, UploadGeoJSONRequest request, ActionListener<UploadGeoJSONResponse> actionListener) {
+        // update metric api count
```

Comment:
nit: do we need this comment? Code is pretty straightforward and self explanatory

Reply:
Ack